test/Test.hs

```diff
@@ -24,41 +22,12 @@ main = do
     removeDirectoryRecursive ".smuggler"
 
     if result
-        then successMessage "Success"
-        else failureMessage "Some tests failed" >> exitFailure
+        then putStrLn "Success"
+        else putStrLn "Fail" >> exitFailure
   where
     compareWithGolden :: FilePath -> IO Bool
     compareWithGolden f = do
         let (name, _) = splitExtension f
         goldenContent <- readFile $ name <.> "golden"
         testContent   <- readFile f
-        if goldenContent == testContent
-          then pure True
-          else do
-            failureMessage $ "\nTest " ++ name ++ " failed:"
-            False <$ outputFailure testContent goldenContent
-
-    outputFailure :: String -> String -> IO ()
-    outputFailure given expected = do
-      boldMessage "Expected:"
-      putStrLn (importStatementsOnly expected)
-      boldMessage "But got:"
-      putStrLn (importStatementsOnly given)
-
-successCode, failureCode, resetCode :: [SGR]
```

Comment:
This is strange that all these functions are missing now :thinking: Probably some issues during conflicts resolution